backend/src/packages/users/libs/enums/table-columns/user-details-table-column.enum.ts

```diff
@@ -0,0 +1,6 @@
+const UserDetailsTableColumn = {
```

Comment:
let's remove table columns, for me sounds like to much

Reply:
@Xxandder we had a discussion with other coaches, and decided on leaving UserDetailsTableColumn and so on, but remove CommonColumns and duplicate column properties where it's needed